### PR TITLE
Adds New Landing Page

### PR DESCRIPTION
**Pull request by AlexCatarino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,229 @@
...
+<p align="center">
+<picture >
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/">
+  <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/">
+  <img alt="introduction" width="40%">
+</picture>
+<p>
+
+[Coinbase](https://www.coinbase.com/) was founded by Brian Armstrong and Fred Ehrsam in 2012 with the goal to "increase economic freedom in the world". Coinbase Pro provides access to trading Crypto for clients in over [100 countries](https://www.coinbase.com/places) with no minimum deposit. Coinbase Pro also provides a self-hosted Crypto wallet, a Visa debit rewards card, and Bitcoin collateral-backed lines of credit.
+
```
Comment (DerekMelchin) [inline README.md:24]: Missing link to docs

File: README.md [hunk 2]
```diff
@@ -1,10 +1,229 @@
...
+  You can deploy using a visual interface in the QuantConnect Cloud. For more information see the [QuantConnect-CoinbasePro Integration Page](https://www.quantconnect.com/brokerages/coinbasepro).
+
+  In the QuantConnect Cloud Platform you can harness the QuantConnect Live Data Feed. For most users this is substantially cheaper and easier than self-hosting.
+  
+### Deploying Coinbase Pro with LEAN CLI
+
+Follow these steps to start local live trading with the Coinbase Pro brokerage:
+
+1.  Open a terminal in your [CLI root directory](https://www.quantconnect.com/docs/v2/lean-cli/initialization/directory-structure#02-lean-init).
+2.  Run lean live "`<projectName>`" to start a live deployment wizard for the project in ./`<projectName>` and then enter the brokerage number.
```
Comment (DerekMelchin) [inline README.md:38]: I think
lean live "`<projectName>`"
should be
`lean live "<projectName>"`

File: README.md [hunk 3]
```diff
@@ -1,10 +1,229 @@
...
+    4. Bitfinex
+    5. Coinbase Pro
+    6. Binance
+    7. Zerodha
+    8. Samco
+    9. Terminal Link
+    10. Trading Technologies
+    11. Kraken
+    12. FTX
+    13. IQFeed
```
Comment (DerekMelchin) [inline README.md:112]: Should IQFeed be here?

File: README.md [hunk 4]
```diff
@@ -1,10 +1,229 @@
...
+    14. Polygon Data Feed
+    15. Custom Data Only
+  
+        To enter multiple options, separate them with comma:
+    ```
+
+If you select Polygon Data Feed, see [Polygon](https://www.quantconnect.com/docs/v2/lean-cli/live-trading/other-data-feeds/polygon) for set up instructions.
+
+1. View the result in the `<projectName>`/live/`<timestamp>` directory. Results are stored in real-time in JSON format. You can save results to a different directory by providing the --output `<path>` option in step 2.
```
Comment (DerekMelchin) [inline README.md:121]: Should be #7

And I think
`<projectName>`/live/`<timestamp>`
should be
`<projectName>/live/<timestamp>`

and 
--output `<path>`
should be 
`--output <path>`

File: README.md [hunk 5]
```diff
@@ -1,10 +1,229 @@
...
+
+If you already have a live environment configured in your [Lean configuration file](https://www.quantconnect.com/docs/v2/lean-cli/initialization/configuration#03-Lean-Configuration), you can skip the interactive wizard by providing the --environment `<value>` option in step 2. The value of this option must be the name of an environment which has live-mode set to true.
```
Comment (DerekMelchin) [inline README.md:123]: --environment `<value>`
should be
`--environment <value>`

live-mode
should be
`live-mode`

**Review by DerekMelchin — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AlexCatarino) [reply README.md:112]: I think so because it's a valid option, and `lean-cli` will display it.

**Pull request merged**